### PR TITLE
Add update & delete JS object properties

### PR DESCRIPTION
**Pull request by avishekjana**

### CODE REVIEW

File: numbers-booleans-objects/add-update-delete-objects/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Script
+
+We've already learned that, the **properties** are the most important part of any JavaScript object. Properties are the values associated with an object.
+
+And in this lesson, we will learn to add, update and delete properties of an Javascript Object.
```
Comment (mjzac) [inline numbers-booleans-objects/add-update-delete-objects/README.md:5]:
```suggestion
And in this lesson, we will learn to add, update and delete properties of a JavaScript Object.
```

File: numbers-booleans-objects/add-update-delete-objects/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### Adding New Properties
+There’re a few ways to add properties to an object in JavaScript. You can add new properties to an existing object by simply giving it a value.
+
+Assume that you've a a *employee* object that already exists. One way is to add a property to this  *employee* object is, by using the **dot notation**:
```
Comment (mjzac) [inline numbers-booleans-objects/add-update-delete-objects/README.md:10]:
```suggestion
Assume that you've an *employee* object that already exists. One way to add a property to this  *employee* object is, by using the **dot notation**:
```

File: numbers-booleans-objects/add-update-delete-objects/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+We added the *department* property to the *employee* object above with value **Software**.
+
+We can also add a property by using the bracket notation:
+
+```js
+employee['department'] = "Software";
+```
+
+It does the same thing as the previous example, but we can have invalid property identifiers in the string.
```
Comment (mjzac) [inline numbers-booleans-objects/add-update-delete-objects/README.md:24]:
```suggestion
It does the same thing as the previous example, but we can have property identifiers with hyphen or space with this notation.
```

File: numbers-booleans-objects/add-update-delete-objects/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+const employee = {
+  firstName: "John",
+  lastName: "Doe",
+  salary: 50000,
+  department: "Software"
+};
+
+delete employee.department;
+```
+This `delete` keyword deletes both the value of the property and the property itself. And, after deletion, the property cannot be used.
```
Comment (mjzac) [inline numbers-booleans-objects/add-update-delete-objects/README.md:60]:
```suggestion
This `delete` keyword deletes both the value of the property and the property itself. And, after deletion, accessing the property will return `undefined`.
```

**Review by mjzac — CHANGES_REQUESTED** [review]

File: numbers-booleans-objects/add-update-delete-objects/README.md (hunk 4 above)
Comment (avishekjana) [reply numbers-booleans-objects/add-update-delete-objects/README.md:58]: Resolved

File: numbers-booleans-objects/add-update-delete-objects/README.md (hunk 3 above)
Comment (avishekjana) [reply numbers-booleans-objects/add-update-delete-objects/README.md:23]: Resolved

File: numbers-booleans-objects/add-update-delete-objects/README.md (hunk 2 above)
Comment (avishekjana) [reply numbers-booleans-objects/add-update-delete-objects/README.md:9]: Resolved

**Pull request merged**